pkg/v1/internet_time.go

```diff
@@ -0,0 +1,80 @@
+package v1
+
+import (
+	"math"
+	"time"
+)
+
+var (
+	defaultAlgorithm       = TotalSeconds
```

Comment:
Will need to discuss if this is the desired default algorithm or not. The nanosecond algo won't match wikipedia.